content/docs/upgrading/upgrade.md

```diff
@@ -4,11 +4,67 @@ description = "Upgrading your deployment to a later version of Kubeflow"
 weight = 30
 +++
 
-Until version 1.0, Kubeflow makes no promises of backwards compatibility or 
-upgradeability. There is **no clean upgrade path** to the latest version of 
-Kubeflow ({{% kf-latest-version %}}).
+Upgrading your Kubeflow deployment is supported if your deployment is v0.7.0 or later.
 
-Nonetheless, here are some instructions for updating your deployment:
+Prerequisites:
+
+* Download the latest kfctl binary from the
+  [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/tag/{{% kf-latest-version %}}).
+
+* Ensure that your kubeflow namespace is annotated with the following label:
+```
+  labels:
+    control-plane: kubeflow
+```
+* You must have a local Kubeflow application directory matching your current
+   deployment.
+
+Upgrade instructions:
+
+1. Create an upgrade spec in the parent directory of the Kubeflow application. An example is provided
```

Comment:
We use the environment variable `${KF_DIR}` to indicate the full path to the user's Kubeflow application directory.

For example:
https://deploy-preview-1455--competent-brattain-de2d6d.netlify.com/docs/started/k8s/kfctl-k8s-istio/

We should use the same terminology in the upgrade guide as we've used in all the deployment guides, to help people understand what we mean here. For example, when reading this paragraph, I'm not sure if the file should be in the KF_DIR or the parent of the KF_DIR.